politeiad/backend/tlogbe/tlog.go

```diff
@@ -181,25 +181,28 @@ func treePointerExists(r recordIndex) bool {
 	return r.TreePointer > 0
 }
 
-// blobIsEncrypted returns whether the provided blob has been prefixed with an
+// BlobIsEncrypted returns whether the provided blob has been prefixed with an
 // sbox header, indicating that it is an encrypted blob.
-func blobIsEncrypted(b []byte) bool {
+func BlobIsEncrypted(b []byte) bool {
 	return bytes.HasPrefix(b, []byte("sbox"))
 }
 
-func leafIsRecordIndex(l *trillian.LogLeaf) bool {
+// LeafIsRecordIndex checks if the leaf is a record index,
+func LeafIsRecordIndex(l *trillian.LogLeaf) bool {
 	return bytes.HasPrefix(l.ExtraData, []byte(keyPrefixRecordIndex))
 }
 
-func leafIsRecordContent(l *trillian.LogLeaf) bool {
+// LeafIsRecordContent checks if the leaf is a record content.
+func LeafIsRecordContent(l *trillian.LogLeaf) bool {
```

Comment:
Why do you need to export these functions?

Reply:
You mean I should rewrite them? Guess it may be cleaner since it's one line of code, so there's no need to export them